bifromq-session-dict/bifromq-session-dict-client/src/main/java/com/baidu/bifromq/sessiondict/client/SessionDictClient.java

```diff
@@ -50,17 +53,16 @@ final class SessionDictClient implements ISessionDictClient {
         regPipeline = Caffeine.newBuilder()
             .weakValues()
             .executor(MoreExecutors.directExecutor())
-            .removalListener(
-                (RemovalListener<String, IRPCClient.IMessageStream<Quit, Session>>) (key, value, cause) -> {
-                    if (value != null) {
-                        value.close();
-                    }
-                })
-            .build(registerKey -> rpcClient.createMessageStream("",
-                null,
-                registerKey,
-                Collections.emptyMap(),
-                SessionDictServiceGrpc.getDictMethod()));
+            .build(registerKey -> {
```

Comment:
1. why coupling cleaner logic with cache building logic?
2. add UT to assert the cleaner is work as expected

Reply:
fixed